src/components/DataAccessLayer.ts

```diff
@@ -1,16 +1,32 @@
-import { resources } from "@/resources";
+import { resources as staticResources } from "@/resources";
```

Comment:
why `as`? I don't see an overriding function other than the returning value. Let's try to avoid giving alias especially this early in the repo.

Reply:
fixed